__tests__/classes/CanvasRenderingContext2D.fillStyle.js

```diff
@@ -11,12 +11,12 @@ beforeEach(() => {
 describe('fillStyle', () => {
   it('should parse a css color string \'blue\'', () => {
     ctx.fillStyle = 'blue';
-    expect(ctx.fillStyle).toBe('#00f');
+    expect(ctx.fillStyle).toBe('#0000ff');
```

Comment:
Please assert that the fillStyle returns `"#00f"` here like the browser does instead of `"#0000ff"`. I believe that you can use the `"short"` parameter when using `moo-color`

Reply:
From the [spec](https://www.w3.org/TR/2dcontext/#serialization-of-a-color):
> if it has alpha equal to 1.0, then the string is a lowercase six-digit hex value, prefixed with a "#" character
> Otherwise, the color value has alpha less than 1.0, and the string is the color value in the CSS rgba() functional-notation format
Testing it in firefox, it also returns it in 6-digit notation rather than the shortened form

Reply:
Yep! For some reason, this spec must have changed, or I overlooked it.